plugins/inventory/nmap.py

```diff
@@ -30,10 +30,18 @@
         address:
             description: Network IP or range of IPs to scan, you can use a simple range (10.2.2.15-25) or CIDR notation.
             required: true
+            env:
+              - name: ANSIBLE_NMAP_ADDRESS
+                version_added: 6.6.0
         exclude:
-            description: list of addresses to exclude
+            description:
+              - List of addresses to exclude.
+              - For example C(10.2.2.15-25) or C(10.2.2.15,10.2.2.16).
             type: list
             elements: string
+            env:
+              - name: ANSIBLE_NMAP_EXCLUDE
+                version_added: 6.6.0
         port:
             description:
                 - Only scan specific port or port range (C(-p)).
```

Comment:
I only now noticed that the YAML in other parts of the documentation is indented differently (lists have an indent of four, not two). Would you mind adjusting the indentation in the newly added/modified parts?

I think besides that this PR looks good. If nobody objects to this change, I'll merge it in ~a week.

Reply:
Thank you for answer.
I can't change indent for `env` because this will broken `yaml` structure and environment variables won't be read from host environment. I can change indent for `name_of_env_variable`. 

You can check it in the [opennebula](https://github.com/ansible-collections/community.general/blob/main/plugins/inventory/opennebula.py#L43-L44) or [cobbler](https://github.com/ansible-collections/community.general/blob/main/plugins/inventory/cobbler.py) dynamic inventory plugins. I found examples where there are `two` and `four` spaces. I think we should use `two` spaces everywhere in the future.
what do you think about it?

Reply:
`env` is already indented by four spaces, so that's not what you were being asked to change. While it might be nice to make the indentation consistent across all plugins, for now we just try to keep it consistent within a particular plugin. This one uses four spaces, so your additions should also use four (to preserve visual alignment of elements at the same level within the structure.)

Reply:
hello @flowerysong 

thank you for your answer.
No problem. I fixed it.